client/my-sites/themes/thanks-modal.jsx

```diff
@@ -222,12 +252,18 @@ class ThanksModal extends Component {
 export default connect(
 	state => {
 		const siteId = getSelectedSiteId( state );
+		const siteUrl = get( getSelectedSite( state ), 'URL', null );
```

Comment:
Oh, just remembered we have this: https://github.com/Automattic/wp-calypso/blob/master/client/state/selectors/get-site-url.js

Reply:
nice. using that now